zstacklib/zstacklib/utils/ovn.py

```
@@ -202,6 +228,23 @@ def restoreNicDriver(pciAddressList):
 
+class ControllerCtl(object):
+    def __init__(self):
+        pass
+
+    @bash.in_bash
+    def resetOvnControllerClusterIndex(self):
+        ret = 0
+        try:
+            cmd = '{cmd} -t ovn-controller sb-cluster-state-reset'.format(cmd=AppCtlBin)
+            ret = bash.bash_r(cmd)
+        except Exception as err:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)